webapp/packages/core-theming/src/themes.ts

```diff
@@ -1,31 +1,44 @@
 /*
  * CloudBeaver - Cloud Database Manager
- * Copyright (C) 2020-2024 DBeaver Corp and others
+ * Copyright (C) 2020-2025 DBeaver Corp and others
  *
  * Licensed under the Apache License, Version 2.0.
  * you may not use this file except in compliance with the License.
  */
 import type { ITheme } from './ThemeService.js';
+import type { ClassCollection } from './themeUtils.js';
 
 const emptyTheme = {};
 
+export enum THEME_ID {
+  LIGHT = 'light',
+  DARK = 'dark',
+  SYSTEM = 'system',
+}
```

Comment:
The theme ID should be just a string, but themes themselves should have type: 'dark' or 'light'
So, in the `ITheme`, you need to add the `type` property